### ISSUE TITLE
Tabs: Document Simple Opacity Transition?

### ISSUE THREAD

**Issue by karlhorky** [reach/reach-ui#151]
Hi, thanks for the project! Really respect the attention given to accessibility here.

I've been playing around with simple opacity transitions between `TabPanel`s by styling the `[hidden]` attribute using `display: block` and `position: absolute`. Not sure if there are drawbacks yet to this pattern.

Would this be something interesting for the docs? I could write a quick section up.

Or would it make sense to add a feature to expose more control over the switching of the elements?

**Comment by karlhorky** [comment]
@bradwestfall would this example be something that is interesting for the project? I guess the stalebot was faster than any replies here...

**Comment by bradwestfall** [comment]
@chancestrickland would be a better person to ask, he's taking the big lead on catching up with issues on this project. I was just helping him by labeling a bunch of stuff

**Comment by chancestrickland** [comment]
@karlhorky Thanks for your interest, and sorry I was too slow to respond. I'm happy to take a look at this if you want to add your idea in the `examples` directory. Hard to say just based on your description what the drawbacks may be, but the more example variations we have the better. Happy to review what you come up with and work with you on getting it into the docs once we get a PR.

**Comment by karlhorky** [comment]
Ok, I'll add this to my backlog. Can we leave this issue open for visibility for anyone else who is looking for this / wants to implement? I guess from the difficulty level it's also a "good first issue".

Re: Drawbacks, I was thinking of maybe if I was actually doing something that is not great for accessibility with those styles. Not sure about that though.

One issue that I have found in the meantime is that `position: absolute`, `opacity` and `display: block` on their own is that the hidden tabs would prevent selecting content from other tabs. So I also added a `pointer-events: none` to the hidden tabs.
